docs/Developer-Guide.md

```diff
@@ -219,15 +219,36 @@ rustup target add x86_64-unknown-linux-musl
 sudo ln -s /usr/bin/g++ /bin/musl-g++
 ```
 
+The agent is built with seccomp capability by default.
```

Comment:
`seccomp`

Reply:
This line becomes false with the changes you submit next, since the default is not to have the two required variables set.

Reply:
I removed the changes from `Makefile`.